src/main/java/com/exasol/adapter/dialects/postgresql/PostgreSQLSqlDialect.java

```diff
@@ -145,6 +146,12 @@ public NullSorting getDefaultNullSorting() {
         return NullSorting.NULLS_SORTED_AT_END;
     }
 
+    @Override
+    // https://www.postgresql.org/docs/9.2/sql-syntax-lexical.html
+    public String getStringLiteral(final String value) {
+        return super.quoteLiteralStringWithSingleQuote(value);
```

Comment:
This should work with all PostgreSQL versions and settings:
```suggestion
        if (value == null) {
            return "NULL";
        } else {
            return "E'" + value.replace("\\", "\\\\").replace("'", "''") + "'";
        }
```